tests/integrationtest/t/globalindex/ddl.test

```diff
@@ -55,20 +28,30 @@ CREATE TABLE `t` (
 PARTITION BY HASH (`b`) PARTITIONS 3;
 show create table t;
 drop table t;
+
 create table t (a int, b int, unique key (a)) partition by hash(a) partitions 3;
 -- error 8264
 alter table t partition by hash(b) partitions 3;
 alter table t partition by hash(b) partitions 3 UPDATE INDEXES (a GLOBAL);
+alter table t add index idxOK (b) global;
+alter table t add unique index idxOK2 (a) global;
+alter table t add unique index idxOK3 (b) global;
+create index idxOK4 on t (b) global;
+create unique index idxOK5 on t (a) global;
+create unique index idxOK6 on t (b) global;
+
```

Comment:
Could you please also add tests for repartitioning a table, where one can switch between GLOBAL and NON-GLOBAL non-unique index? Like:
```
alter table t partition by key(b) partitions 5 UPDATE INDEXES (idxOK LOCAL, idxOK2 GLOBAL, idxOK3 LOCAL);
```
Ideally so that non-unique indexes will test local->local, local->global, global->local and nglobal->global  (both implicitly and explicitly, where possible)

Reply:
Done